src/main/java/com/epam/lowcost/controller/FlightController.java

```diff
@@ -28,10 +28,24 @@ public String getAllFlights(Model model) {
     @GetMapping
     public String findFlightById(@RequestParam Long id, Model model) {
         model.addAttribute("flight", flightService.getById(id));
-        return "flights";
+        return "flightSettings";
+    }
+
+    @GetMapping (value = "/newTicket")
```

Comment:
@IliaStepanov reminded whole team that we use `-` instead of camelCase. @IliaStepanov, please tell about this again in Telegram chat

Reply:
ok